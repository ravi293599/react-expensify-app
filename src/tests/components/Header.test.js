import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
//import ShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../components/Header';

test('should render Header correctly', () => {
	const wrapper = shallow(<Header />);
	expect(toJSON(wrapper)).toMatchSnapshot();
/*	const renderer = new ShallowRenderer();
	renderer.render(<Header />);
	expect(renderer.getRenderOutput()).toMatchSnapshot();*/
});