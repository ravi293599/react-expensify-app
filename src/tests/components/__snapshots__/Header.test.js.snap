// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    AddExpense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/edit"
  >
    EditExpense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
